File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        char[] UpDown=my_string.toCharArray();
        for(int i=0; i<UpDown.length; i++){
            if(UpDown[i]>=65 && UpDown[i]<=90){//대문자라면? UpDown[i]>='A' UpDown[i]<='Z'작성도 가능
                UpDown[i]=(char)(UpDown[i]+32); //소문자로 변환
            }
            else if(UpDown[i]>=97  && UpDown[i]<=122){ //소문자라면? UpDown>='a' &&  UpDown[i]<='z'
                UpDown[i]=(char)(UpDown[i]-32);
            }
        }
        String answer = String.valueOf(UpDown);
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    word=my_string.split('');
    for(let i=0; i<word.length; i++){
        if(word[i]>='A' && word[i]<='Z'){
            word[i]=String.fromCharCode(word[i].charCodeAt(0)+32);
        }
        else if(word[i]>='a' && word[i]<='z'){
            word[i]=String.fromCharCode(word[i].charCodeAt(0)-32);
        }
    }
    let answer = word.join(''); // 배열을 다시 문자열로 변환
    return answer;
}